src/views/Variants/components/PageContent/index.tsx

```diff
@@ -51,21 +54,40 @@ const PageContent = ({ variantMapping }: OwnProps) => {
   const [variantQueryConfig, setVariantQueryConfig] = useState(DEFAULT_QUERY_CONFIG);
   const variantResolvedSqon = resolveSyntheticSqon(queryList, activeQuery);
 
-  const variantResults = useVariant({
-    first: variantQueryConfig.size,
-    offset: variantQueryConfig.size * (variantQueryConfig.pageIndex - 1),
-    sqon: variantResolvedSqon,
-    sort: isEmpty(variantQueryConfig.sort)
-      ? [
-          { field: 'max_impact_score', order: 'desc' },
-          { field: 'hgvsg', order: 'asc' },
-        ]
-      : variantQueryConfig.sort,
-  });
+  const variantResults = useVariant(
+    {
+      first: variantQueryConfig.size,
+      offset: DEFAULT_PAGE_INDEX,
+      searchAfter: variantQueryConfig.searchAfter,
+      sqon: variantResolvedSqon,
+      sort: tieBreaker({
+        sort: variantQueryConfig.sort,
+        defaultSort: DEFAULT_SORT_QUERY,
+        field: 'locus',
+        order: SortDirection.Asc,
+      }),
+    },
+    variantQueryConfig.operations,
+  );
+
+  useEffect(() => {
```

Comment:
y'a 2 useEffect pour trigger 2 `setVariantQueryConfig`, ca peut etre simplifier pour etre comprehensible et aussi éviter un double render ?

Reply:
C'est plutôt conseillé par React d'utiliser plusieurs useEffect selon ce qu'on veut faire https://reactjs.org/docs/hooks-effect.html#tip-use-multiple-effects-to-separate-concerns

Ici le premier sert uniquement a stocker le flag pour la première page.
Le second gère lorsque la query bar change. 

Leur séparation permet de gérer ces deux cas de façon isolé.